Correctly resolve test bundle

When adding this to Kibana, I see a lot of errors like this:

```
/repos/kibana/src/core_plugins/kibana/public/dashboard/__tests__/panel.js
  2:8   error  No default export found in module                                 import/default
  5:29  error  Unable to resolve path to module 'test_utils/no_digest_promises'  import/no-unresolved
  6:25  error  Unable to resolve path to module 'fixtures/mock_ui_state'         import/no-unresolved
```

Several hundred are related to things from `test_utils` and `fixtures`.

--- a/index.js
+++ b/index.js
@@ -152,8 +152,8 @@ function resolveKibanaModuleImport(source, kibanaPath) {
   ];
 
   // clean the source
-  // strip off leading `ui/` if its there
-  const baseSource = source.replace(/^ui\//, '');
+  // strip off leading prefix, if its there (ui, fixtures, test_utils, test_harness)
+  const baseSource = source.replace(/^(ui|fixtures|test_utils|test_harness)\//, '');
   let resolved = { found: false };
   checkPaths.forEach(function (checkPath) {
     if (!resolved.found) {

